rpa-manager/modules/ROOT/pages/processautomation-manage.adoc

```diff
@@ -242,8 +250,51 @@ Open the xref:processoperations-financeanalysis.adoc[*Finance Analysis* view] to
 
 Once it's ready, transition the project to the next phase of its life-cycle. You can always transition back to change the project's design or implementation.
 
+In the **Design** phase, you can do the following:
+
+* Create or change a process model in RPA Manager.
++
+Already implemented activities don’t lose their implementation if you don’t delete them. 
+* Transfer the automation containing the new or changed models to the Build Phase.
+
+In the **Build** phase, in RPA Builder, you can do the following:
+
+* Download the model from RPA Manager.
+* Change the model.
+* Implement the activities.
+* If you want to version workflows, create and reuse activities in the Activity Library.
+* Upload the changed and implemented model to the repository in RPA Manager. 
++
+There is always only one version of a process in the repository in RPA Manager. There might be another local version in RPA Builder. If you accidentally deleted an already implemented Activity, you might still be able to find it in RPA Builder - as long as you do not overwrite the model by downloading.
+* Transfer the automation back to the Design phase.
++
+Already implemented activities don’t lose their implementation if you don’t delete them. 
+* Transfer the automation on to the Test phase.
+
+In the **Test** phase, you can do the following:
+
+* Test the implemented process model in different configurations. 
+* Transfer the automation back to Build phase to change the implementation.
++
+You can reuse or edit the test run configurations when you transfer the automation to the Test phase again. 
+* When the tests are successful, transfer the automation to the Production phase.
++
+If the project has been transferred at least once to the production phase and deployed to an RPA Bot for processing, it can no longer be deleted.
+
+In the *Production* phase, you can do the following. 
+
+* Create and deploy a **Scheduled Run Configuration** to run the process automatically at defined times. 
+* Create and publish an **Invocable Run Configuration** to use it within other Salesforce products.
+* Create an **On-Demand Run Configuration** to start the process run manually.
```

Comment:
Same issue. I'm not sure how to rewrite this.

Maybe "Click *On-Demand Run Configuration* to...."

Reply:
Is "process run" an action?